### PR TITLE
Don't crash when we get unexpected pointer events.  Just log them as errors.

### PR DESCRIPTION
**Pull request by apwilson**

### CODE REVIEW

File: content_handler/runtime_holder.cc [hunk 1]
```diff
@@ -570,9 +570,22 @@ void RuntimeHolder::OnEvent(mozart::InputEventPtr event,
           pointer_data.change = blink::PointerData::Change::kHover;
         break;
       case blink::PointerData::Change::kAdd:
+        if (down_pointers_.count(pointer_data.device) != 0) {
+          FXL_DLOG(ERROR) << "Received add"
+                          << " event for pointer that is currently down!";
```
Comment (abarth) [inline content_handler/runtime_holder.cc]: Maybe combine these two strings.  I'd also remove the ! and use a .  (Same comment below.)

**Comment by abarth** [comment]
LGTM

**Comment by abarth** [comment]
LGTM

**Comment by chinmaygarde** [comment]
Can we land this please?

**Comment by abarth** [comment]
Looks like there's a formatting error.  I can fixenate that if @apwilson likes.

**Comment by abarth** [comment]
Closing in favor of https://github.com/flutter/engine/pull/4230

**Pull request closed without merging**